site/content/entry/yuzu-progress-report-jul-2023/index.md

```diff
@@ -0,0 +1,201 @@
++++
+date = "2023-08-06T06:45:00+05:30"
+title = "Progress Report July 2023"
+author = "CaptV0rt3x"
+coauthor = "GoldenX86"
+forum = 0
++++
+
+Hey there yuz-ers!
+Welcome back to our monthly report on all the improved features, performance gains, and bug fixes we made.
+Let's jump right in!
+
+<!--more--> 
+
+## Aliens and ghosts are real! And they run on Unreal Engine
+
+The wait was long but worth it, another game in the charming Pikmin franchise is finally arriving on Earth, just in time to join the wave of UFO declarations!
+
+{{< imgs
+	"./pk1.png| Imagine a cross-over with Toy Story, it would explain The Claw! (Pikmin 4)"
+  >}}
+
+This time, `Pikmin 4` decided to use Unreal Engine 4, which guarantees good performance and quality on the Switch.
+This engine also guarantees headaches implementing its particular interest in sparse textures and the stability problems it introduces to NVIDIA GPU users running Vulkan, as previous Unreal Engine 4 games have demonstrated.
+
+As you can expect, there’s a lot of work to do to improve compatibility on games running this particular popular engine.
+
+[byte[]](https://github.com/liamwhite) took the first blow, by finding which part of the texture cache code caused a regression {{< gh-hovercard "11093" "affecting stability" >}} strong enough to cause device losses (the driver cutting off the GPU from rendering, resulting in an emulator crash) on NVIDIA GPUs.
+
+{{< imgs
+	"./pk2.png| Moss seems tired (Pikmin 4)"
+  >}}
+
+[emufan](https://github.com/GPUCode) followed up, he identified that hardcoding the total number of texture buffers to `16` was causing crashes in some Unreal Engine 4 games and {{< gh-hovercard "11098" "bumped up the count" >}} to `32`.
+
+{{< single-title-imgs
+    "Good thing this game didn't cross paths with Pikmin (Master Detective Archives: RAIN CODE)"
+    "./rain1.png"
+    "./rain2.png"
+    >}}
+
+This not only has helped `Pikmin 4 but also `Master Detective Archives: RAIN CODE`, another recent release using this same engine.
+
+{{< imgs
+	"./rain3.png| Yeah, good name for a train, I'll board it in Dracula Station (Master Detective Archives: RAIN CODE)"
+  >}}
+
+# Per-game configurations - Reimagined!
+
+If you're not a new yuz-er, it is very likely that you've seen and used per-game configurations.
+But incase you haven't yet, per-game configurations are a way to set game specific settings without having to redo your settings for every game you start.
+Imagine if few games require GPU accuracy to be "HIGH" instead of "NORMAL" or if some games work best with "OpenGL" over "Vulkan", with per-game configurations you can easily override these specific settings for just those games.
```

Comment:
```suggestion
For the games which require GPU accuracy to be high instead of normal, or for games which work best with OpenGL over"Vulkan, per-game configurations allow you to easily override these specific settings for just those games.
```